### ISSUE TITLE
only one listener per command?

### ISSUE THREAD

**Issue by formvoltron** [escamoteur/flutter_command#10]
This seems like it should work, but i must be not understanding something.

I have one command and a functional_listener on that command to re-render the results as something else.    

Only the functional_listener gets called and the watchX on the command does not trigger.

When I remove the functional_listener, the watchX on the command works, as expected.

So it seems that a command can have only one listener?  How about allowing N listeners per command?   Allowing only one listener per command creates side - effect bugs that will be hard to track down.

My other option seems to be to create a special class to bundle all the UI specific versions of a command response, but this is introducing boilerplate.

**Comment by Abhilash-Chandran** [comment]
@formvoltron Thanks for reporting an issue here 👍. 

Could you please share a small piece of code about what you are trying? Would help us look into it quickly. I am sure we can have multiple listeners associated to a single command. Commands are nothing but ValueListenables and can already have N listeners hooked up to it.

**Comment by formvoltron** [comment]
@Abhilash-Chandran Hi Abhilash.  Thank you for a speedy reply!

Well, I created a minimum version that produces the error... but when I made it, I had no error!  Then I went back to my original program... and it works there too!

Bizarre.  It has been some years since I was programming... I must have just missed something. 

However.  I did find some strange behavior.  I will create a new bug for that.
